daemon/targetclid

```diff
@@ -164,9 +164,9 @@ class TargetCLI:
 
                 with open('/tmp/data.txt', 'r') as f:
                     output = f.read()
+                    var = struct.pack('i', len(output))
+                    connection.sendall(var)    # length of string
```

Comment:
May be club this patch with patch with subject "Only return response to targetcli when bytes present"